frontend/app/MakeRoom/page.tsx

```diff
@@ -0,0 +1,74 @@
+'use client'
+import React, { useState } from "react";
+import "./MakeRoom.css";
+
+const MakeRoom: React.FC = () => {
```

Comment:
Could you branch off my branch: https://github.com/ll886/skribbl.io/pull/5/files

Update the form to send a POST request to the `/api/rooms/` endpoint to create a room. 

The body will look something like this:
```
{
    "drawTime": <number>,
    "numRounds": <number>,
}
```

on my branch, there should be a `config.tsx` that exports a `serverUrl` variable that you can use for your `fetch` request

Reply:
so I assume you only want draw time and number of rounds as the options.